include/kernels/gauge_stout.cuh

```diff
@@ -146,7 +148,7 @@ namespace quda
       // Compute Omega_{mu}=[Sum_{mu neq nu}rho_{mu,nu}C_{mu,nu}]*U_{mu}^dag
       //-------------------------------------------------------------------
       // Compute \rho * staple_coeff * S - \rho * rectangle_coeff * R
-      Q = ((arg.staple_coeff * Stap) - (arg.rectangle_coeff * Rect)) * conj(U);
+      Q = ((arg.staple_coeff * static_cast<const Link &>(Stap)) - (arg.rectangle_coeff * static_cast<const Link &>(Rect))) * conj(U);
```

Comment:
Pedantic, but should the `static_cast` be a `reinterpret_cast`? (Also, as a point of future work, is there a way to get rid of the need for the cast outside of a full expression template engine? I can see how multiplying by a scalar, on face value, requires the explicit cast.)

Reply:
discussed offline: `static_cast` is correct here.